app/src/main/java/com/example/habitsmasher/DaysTracker.java

```diff
@@ -0,0 +1,261 @@
+package com.example.habitsmasher;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Set;
+
+public class DaysTracker {
+    /**
+     * This class is used to represent the days of the week a habit takes place, and contains all functionality in
+     * storing, querying, modifying, and returning what days of the week a habit takes place. This class does not currently handle
+     * storing the days in a database, but is capable of taking in information stored in a database and converting to a DaysTracker object.
+     *
+     * @author: Cameron Matthew
+     * @Version: 1.0
+     * @see Habit
+     */
+
+    //instance variables to represent days of the week
+    private ArrayList<Boolean> _days;
+
+
+    /**
+     * Creates a DaysTracker object with all values set to false. This constructor is the default constructor.
+     */
+    public DaysTracker(){
+        //arraylist of booleans that represent the days of the week. autofilled to 7 bools.
+        _days = new ArrayList<Boolean>(Arrays.asList(new Boolean[7]));
+
+        //fill with all false
+        Collections.fill(_days, Boolean.FALSE);
```

Comment:
So I see we have a helper method called `setFalse()`, why are we not using that here?

Reply:
Fixed, is now being used.